### ISSUE TITLE
Home Assistant: numeric_state entity_id checking complains about anything other than sensor.*

### ISSUE THREAD

**Issue by rccoleman** [keesschollaart81/vscode-home-assistant#441]
I will see if I can fix and test this (I have before), but I'm going to paste the code here in case it's easier for someone else.

The Home Assistant extension (1.8.1?) appears to have added restrictions to the entity name checking for `numeric_state` that are too strict. I have automations that use `input_number.xxx` and `sun.sun` with a `numeric_state` trigger/condition and they work fine, but they result in complaints with the new HA extension. They passed the checks in a previous extension release.

```YAML
- alias: Garage open turn on lights
  trigger:
    - platform: state
      entity_id: binary_sensor.garage_door_sensor
      to: 'off'
  condition:
    condition: numeric_state
    entity_id: sun.sun
    value_template: "{{ state.attributes.elevation }}"
    below: 12
  action:
    - data:
        entity_id:
          - group.garage
      service: homeassistant.turn_on
```

```YAML
- alias: 'Away Timer Expired'
  trigger:
    - platform: numeric_state
      entity_id: input_number.away
      below: 1
  action:
    - entity_id: automation.away_timer
      service: automation.turn_off
```

The naming restriction on the acceptable domains that can be used with `numeric_state` should be loosened to include others that can also return numbers.  `input_number` and `sun.sun` for sure, but there are probably others.

![image](https://user-images.githubusercontent.com/860888/88697017-d7b8e500-d0b8-11ea-86e6-c6242b122c07.png)

![image](https://user-images.githubusercontent.com/860888/88697117-f7e8a400-d0b8-11ea-91df-50450c3dfac7.png)

**Comment by frenck** [comment]
👍  Thanks, that makes sense, will adjust this

**Comment by frenck** [comment]
Fixed in #442, thanks for reporting! 👍
